src/pages/components/list/usage.mdx

```diff
@@ -57,47 +61,259 @@ number or a bullet.
+
+</Column>
+</Row>
+
+<Caption>Unordered and ordered lists with nested items</Caption>
+
+### Placement
+
+List items should be concise, aligning with grid columns to maintain a
+structured layout. Avoid overly long sentences; if content exceeds the available
+space, it should wrap to multiple lines rather than forcing excessive width.
+
+<DoDontRow>
+  <DoDont type="do" caption="Do align list markers flush with the grid.">
+
+![Do align the list vertically for clarity.](images/list-usage-placement-do.png)
+
+  </DoDont>
+  <DoDont type="dont" caption="Do not let list markers hang into the grid gutters.">
+
+![Do not let list markers hang into the grid gutters.](images/list-usage-placement-dont.png)
+
+  </DoDont>
+</DoDontRow>
+## Content
+
+### Main elements
+
+#### List item
+
+- Represents an individual entry within a list.
+- Can contain text, links, or other inline elements.
+- Supports multi-line wrapping for longer content while maintaining alignment.
+- Lists should present simple pieces of information; for complex data, consider
+  using a [data table](/components/data-table/usage) instead.
+- Level 1 list items define the main structure, while Level 2 (nested) list
+  items provide hierarchy or subcategories.
+- Arrange ordered list items logically, such as ranking by importance, highest
+  to lowest values, or in alphabetical/numeric order.
+- Ensure list items are grammatically parallel by maintaining consistent
+  sentence structures (e.g., avoid mixing passive and active voice).
+
+#### Marker
+
+- Markers should be top-aligned with the list item for clarity and consistency.
+- Unordered list level 1 items are marked with dashes and level 2 items are
+  marked with squares.
+- Ordered list level 1 items are prefixed with numbers and level 2 items are
+  prefixed with letters.
+
+### Overflow content
+
+Text in a list should wrap below the counter/marker, keeping it top-aligned with
+the list item to ensure readability. Truncation should be avoided and text
+should instead wrap to multiple lines.
+
+<Row>
+<Column colLg={12}>
+
+![Text wrapping for unordered and ordered lists](images/list-usage-overflow.png)
+
+</Column>
+</Row>
+
+<Caption>Text wrapping for unordered and ordered lists</Caption>
+
+### Further guidance
+
+For further content guidance, see Carbon's
+[Content guidelines](https://carbondesignsystem.com/guidelines/content/overview/).
+
+## Unordered list
+
+Unordered list is a collection of items presented without a specific order. Each
+list item begins with a en dash ("-"), making it ideal for non-sequential
+content.
+
+<Row>
+<Column colLg={12}>
+
+![Example of the unordered list variant in a UI](images/list-usage-unordered-in-context.png)
+
+</Column>
+</Row>
+
+<Caption>Example of the unordered list variant in a UI</Caption>
+
+## Ordered list
+
+Ordered list is a collection of items presented without a specific order. List
+item begins with a number ("1, 2, 3..."), indicating its position, making it
```

Comment:
```suggestion
item begins with a number, indicating its position, making it
```

I think just saying the word "number" here is enough. You could also specify nested items have letter markers.